xcm/src/v3/traits.rs

```diff
@@ -199,6 +205,18 @@ impl From<SendError> for Error {
 
 pub type Result = result::Result<(), Error>;
 
+#[derive(
+	Copy, Clone, Encode, Decode, Eq, PartialEq, Debug, TypeInfo, frame_support::PalletError,
+)]
+pub enum ConversionError {
+	AssetIdLengthExceeded,
+	BadMonikerLength,
+	CannotConvertBlob,
+	CannotConvertGeneralKey,
+	MaxAssetCountExceeded,
+	UnsupportedXcmErrorVariant,
```

Comment:
I think I should use `#[codec(index = $u8)]` here to ensure that all enum variants have a well-defined SCALE encoding? I don't surmise that this enum would grow in the number of variants, but perhaps it's still a good practice to keep regardless?